internal/schema/resource_data.go

```diff
@@ -78,8 +78,14 @@ func (d *ResourceData) IsEmpty(key string) bool {
 	return g.Type == gjson.Null || len(g.Raw) == 0 || g.Raw == "\"\"" || emptyObjectOrArray(g)
 }
 
-func (d *ResourceData) References(key string) []*ResourceData {
-	return d.referencesMap[key]
+func (d *ResourceData) References(keys ...string) []*ResourceData {
```

Comment:
Changed to provide simpler way to fetch more than one reference

Reply:
🙌